src/components/AccountFormFields/index.tsx

```diff
@@ -56,11 +55,16 @@ function AccountFormFields({
 
   // Note: As of 1/22, the password must not contain a period to allow
   // patrons to log into Overdrive for digital reading.
-  const validatePassword = (val) => {
+  const validatePasswordLength = (val) => {
     return (
-      val.length >= minPasswordLength &&
-      val.length <= maxPasswordLength &&
-      val.indexOf(".") === -1
+      (val.length >= minPasswordLength && val.length <= maxPasswordLength) ||
```

Comment:
Do we still need the . period check as the comment above describes?

Reply:
we don't, its an old requirement from overdrive. I can remove it

Reply:
The only thing is all of the translations mention it, I feel like we should keep the check as long as they are there?